cron-jobs/removeUnverifiedUsers.js

```diff
@@ -0,0 +1,40 @@
+const CronJob = require('cron').CronJob
+const tokenService = require('~/services/token')
+const userService = require('~/services/user')
+const tokenNames = require('~/consts/auth')
+
+const EVERY_MIDNIGHT = '0 0 * * *'
+
+const removeUnverifiedUsers = new CronJob(EVERY_MIDNIGHT, () => removeUsersWithUnconfirmedEmail())
+
+const removeUsersWithUnconfirmedEmail = async () => {
+  
+  const usersWithConfirmToken = await tokenService.findTokensWithUsersByParams({ confirmToken: { $ne: null } })
+
+  if(!usersWithConfirmToken?.length){
```

Comment:
Why do we need here '?.' operator?

Reply:
in case if we received null or undefined

Reply:
Removed.